lib/chef-dk/policyfile_services/export_repo.rb

```diff
@@ -85,13 +109,35 @@ def export
 
       private
 
-      def create_repo_structure
-        FileUtils.rm_rf(cookbooks_dir)
-        FileUtils.rm_rf(policyfiles_data_bag_dir)
+      def with_staging_dir
+        p = Process.pid
+        t = Time.new.utc.strftime("%Y%m%d%H%M%S")
+        Dir.mktmpdir("chefdk-export-#{p}-#{t}") do |d|
+          begin
+            @staging_dir = d
```

Comment:
I looked into it and it made everything a lot uglier. In this case I don't think it's that big a deal since everything that depends on `staging_dir` is private, so there's no way to call a public method where it would fail because of `staging_dir` being `nil`. Thoughts?